### DISCUSSION TITLE
Mystery Client

### DISCUSSION THREAD

**Discussion by Indyandie** [hyprwm/Hyprland#2142] [Q&A]
Hello everyone! When I run `hyprctl clients -j`, I'm seeing the following client object.

```json
{
    "mapped": false,
    "hidden": false,
    "at": [0, 0],
    "size": [0, 0],
    "workspace": {
        "id": -1,
        "name": ""
    },
    "floating": false,
    "monitor": -1,
    "class": "",
    "title": "",
    "pid": 0,
    "xwayland": true,
    "pinned": false,
    "fullscreen": false,
    "fullscreenMode": 0,
    "fakeFullscreen": false,
    "grouped": [],
    "swallowing": null
}
```

It's unnamed and weirder it's in monitor `-1`. Any ideas what this is?

Thank you!

**Comment by vaxerski** [comment]
unmanaged, unmapped x11 window

**Reply by Indyandie** [reply]
Thanks @vaxerski! 🙇‍♂️

Any ideas what could cause this, is it normal behavior? Is there a way I can kill it? Or could this have something to do with xwayland?

**Reply by vaxerski** [reply]
normal behavior, welcome to the shitfest that is x11. windows with "mapped": false should be ignored by any and all scripts as they are invisible to the user
